BossMod/Data/ActorState.cs

```diff
@@ -259,15 +259,18 @@ protected override void ExecActor(WorldState ws, Actor actor)
             public override void Write(ReplayRecorder.Output output) => WriteTag(output, Value ? "COM+" : "COM-").EmitActor(InstanceID);
         }
 
-        public event EventHandler<Actor>? ModelStateChanged;
+        public event EventHandler<(Actor, byte, byte, byte)>? ModelStateChanged;
         public class OpModelState : Operation
         {
             public ActorModelState Value;
+            public byte ModelState;
+            public byte AnimState1;
+            public byte AnimState2;
```

Comment:
???

Reply:
fine I will remove it, i wasn't sure if it was needed or not